### ISSUE TITLE
scrolling issues with two SliverMasonryGrid.count in the same CustomScrollView

### ISSUE THREAD

**Issue by moisesbinzie** [letsar/flutter_staggered_grid_view#335]
There are scrolling issues when you add two SliverMasonryGrid.count in the same CustomScrollView Widget

**Comment by kawi15** [comment]
Did you found any solution for this?

**Comment by moisesbinzie** [comment]
No, I didn't. But I used different gridviews from the staggered package to
achieve the layout I wanted. SliverToBox adapter with AlignedGridView.count
and the and the SliverMasonaryGrid.count ..

I am limiting the list of the items being built though.

If you get a solution to this, please let me know

**Comment by kawi15** [comment]
I used another package [https://pub.dev/packages/masonry_grid](https://pub.dev/packages/masonry_grid).
It's has some performance issues but scrolling working properly.

**Comment by moisesbinzie** [comment]
That's great. I hope they fix those issues sooner.

**Comment by PrimakPavel** [comment]
Have the same issue. Please, fix its

**Comment by frankfancode** [comment]
I'm experiencing a scrolling issue on my device. When the second `SliverMasonryGrid` appears and I scroll to the bottom, the `CustomScrollView` automatically scrolls back to the top.

Here's my current workaround: I replaced the first `SliverMasonryGrid` with a `MasonryGridView`, and this resolved the issue.

```dart
SliverToBoxAdapter(
  child: MasonryGridView.count(
    shrinkWrap: true,
    physics: const NeverScrollableScrollPhysics(),
    crossAxisCount: 2,
    padding: EdgeInsets.zero,
    mainAxisSpacing: 8.w,
    crossAxisSpacing: 8.w,
    itemCount: childCount,
    itemBuilder: (BuildContext context, int index) {
      return item();
    }
  ),
)
